rocky85-ci.dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG BPROTAG=latest
+FROM ghcr.io/externpro/buildpro/rocky85-bld:${BPROTAG}
+LABEL maintainer="smanders"
+LABEL org.opencontainers.image.source=https://github.com/externpro/buildpro
```

Comment:
I was assuming now that they are in externpro, we would want these four lines. I do again feel like I am being lazy by making you the maintainer 😆 
Let me know if I need to change something though. I did also add BPROTAG to the yml file

Reply:
this looks great! thanks